CHANGELOG.md

```diff
@@ -5,6 +5,10 @@
 	## __WORK IN PROGRESS__
 	(at the beginning of a new line)
 -->
+
+## __WORK IN PROGRESS__
+* (Avarios) Updatet devcontainer template NGINX : Replaced socket.io with @websocket
```

Comment:
This also needs a migration guide for users who want to modify their existing setups.